src/cosmoz-bottom-bar-focused-context.js

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useState, useMemo } from '@pionjs/pion';
+import { noop } from '@neovici/cosmoz-utils/function';
+
+export const bottomBarFocused = createContext({
```

Comment:
We should drop the `bottomBar` prefix from every name in this file. We are *in* the bottom-bar package, there's no need to prefix things.

```suggestion
export const Focus = createContext({
```

Reply:
I will import `useBottomBarFocusedCtxHandler` in cosmoz-frontend, so I'm keeping the bottomBar prefix for that function?